cargo-apk/README.md

```diff
@@ -51,10 +51,15 @@ apk_name = "myapp"
 runtime_libs = "path/to/libs_folder"
 
 # Defaults to `$HOME/.android/debug.keystore` for the `dev` profile. Will ONLY generate a new
-# debug.keystore if this file does NOT exist.
-# A keystore path is always required on the `release` profile.
+# debug.keystore if this file does NOT exist. A keystore is never auto-generated
+# for other profiles.
+#
+# If specified, the keystore path can be absolute, or relative to the Cargo.toml
+# file. If not specified, the environment variable `CARGO_APK_<PROFILE>_KEYSTORE`
+# environment variable is read. The profile portion follows the same rules as
```

Comment:
Little `environment variable ... environment variable` repeat, and be sure to mention `_KEYSTORE_PASSWORD` :)